lib/components/Button/Button.tsx

```diff
@@ -92,7 +92,7 @@ export const Button = ({
     .trim();
   const spanClassName = `
       ${!disabled ? 'group-hover:underline group-active:no-underline group-focus-visible:no-underline font-poppins' : ''}
-      ${size === 'sm' ? 'py-[10px]' : ''}
+      ${size === 'sm' ? 'py-3' : ''}
```

Comment:
Seem a bit inconsistent to use both logical and ad-hoc values. Also, in default spacing scale, 3 is 12px, so unless overridden it is larger than md. Is this intentional (or maybe we dot use the default scale?)

Reply:
We don't use default values, in Tailwind config, the spacing for scale 3 is 8px. About the ad-hoc values, I personally think that they should be avoided as much as possible, and use configured values instead. I'll bring this up for discussion between UI devs and designers. The UI will become far more consistent and easier to manage if predefined values are used everywhere.